control_panel_api/tests/test_services.py

```diff
@@ -0,0 +1,16 @@
+from unittest.case import TestCase
+
+from control_panel_api import services
+
+
+class ServicesTestCase(TestCase):
+    def test_policy_document_readwrite(self):
```

Comment:
(I know the tests are still work in progress so apologies in advance for commenting on this)

I this the tests should be a bit more in depth as the one in the lambda functions code: https://github.com/ministryofjustice/analytics-platform-ops/blob/master/infra/terraform/modules/data_access/teams/tests/conftest.py

Test the policies are correct is not the nicest thing in the world as these JSON documents are not exactly pretty :D - but at the same time we can protect ourselves from accidental future changes that break them (as that would have the effect of giving the wrong level of access to these S3 buckets to the users with this policies).

Reply:
Ok I can test the entire doc - I was just testing for the addition of that extra part.

Reply:
@aldo

Reply:
I've seen the new tests which check the policy documents 👍